### ISSUE TITLE
Define browsing contexts for transitions

### ISSUE THREAD

**Issue by cvan** [jakearchibald/navigation-transitions#21]
There's a lot covered in https://html.spec.whatwg.org/multipage/browsers.html#nested-browsing-contexts. The spec language ought to just be clarified to ensure that the Transitions lifecycle and ownership models respect these rules.

**Comment by cvan** [comment]
Also, [this document](https://html.spec.whatwg.org/multipage/browsers.html#navigate) is particularly useful to refer to.

**Comment by jakearchibald** [comment]
Agreed. It's probably reasonable to limit transitions to top-level navigations.
